gstatus/libcommand/glustercmd.py

```diff
@@ -106,8 +106,8 @@ def command_thread():
                                                preexec_fn=os.setsid)
 
             stdout, stderr = self.cmdProcess.communicate()
-            self.stdout = stdout.split('\n')[:-1]
-            self.stderr = stderr.split('\n')[:-1]
+            self.stdout = stdout.decode('utf-8').split('\n')[:-1]
```

Comment:
Use `universal_newlines=True` parameter in `Popen` command(Line 101). Decode is not required with that use

Reply:
Done